flake-ctl/src/podman.rs

```diff
@@ -74,13 +75,26 @@ pub fn load(oci: &String) -> i32 {
     /*!
     Call podman load with the provided oci tar file
     !*/
+    let mut container_archive: String = oci.to_string();
+    
     info!("Loading OCI image...");
-    info!("podman load -i {}", oci);
-
+    if !Path::new(oci).exists() {
+        let container_archives = oci.to_owned() + "*";
+        // glob puts things in alpha sorted order which is expected to give
+        // us the highest version of the archive
+        for entry in glob(&container_archives)
+            .expect("Failed to read glob pattern") {
+                match entry {
+                    Ok(path) => container_archive = path.display().to_string(),
+                    Err(_e) => {},
```

Comment:
Please change this to just `Err(_) => { }`, it's a convention in the code that we use `_` only to indicate the information is not used

Reply:
Fixed